README.md

```diff
@@ -2,19 +2,19 @@
 
 A Ruby library to expand shortened URLs.
 
-**Current version:** 1.3.0  
+**Current version:** 1.4.0
```

Comment:
Can you keep the trailing spaces here as they are required for Markdown line breaks?